app/src/main/java/com/keylesspalace/tusky/util/StringUtils.kt

```diff
@@ -3,10 +3,16 @@
 package com.keylesspalace.tusky.util
 
 import android.text.Spanned
+import java.util.regex.Pattern
 import kotlin.random.Random
 
 private const val POSSIBLE_CHARS = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
+const val WORD_BREAK_EXPRESSION = """(^|$|[^\p{L}\p{N}_])"""
+const val WORD_BREAK_FROM_SPACE_EXPRESSION = """(^|$|\s)"""
+const val HASHTAG_EXPRESSION = "([\\w_]*[\\p{Alpha}_][\\w_]*)"
+val hashtagPattern by unsafeLazy { Pattern.compile(HASHTAG_EXPRESSION, Pattern.CASE_INSENSITIVE or Pattern.MULTILINE) }
```

Comment:
I think `unsafeLazy` is unnecessary here since it is a top level property.
`unsafeLazy` is for cases where you don't want class members to be initialized at class creation time but at first access (e.g. to make sure a `Context` is available)